src/index.html

```diff
@@ -4,7 +4,11 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0"
```

Comment:
The viewport meta tag contains 'user-scalable=no', which might not be ideal for accessibility. Consider allowing users to zoom by removing this property to enhance accessibility for users who rely on zooming to read content.